radar/src/radar.rs

```diff
@@ -66,6 +66,31 @@ pub(crate) fn handle_frame(ctx: &CapabilitiesContext, msg: messaging::BrokerMess
 
         let updates = {
             let all_positions = POSITIONS.read().unwrap().clone();
+
+            if all_positions.is_empty() {
+                let entities = ctx.kv().set_intersect(&vec![
+                    format!("decs:{}:tranponder:entities", frame.shard),
```

Comment:
this should be `transponder` - this intersection has a typo in it and will never return any results.

Reply:
Haha, didn't commit that change, my bad there. Current dockerhub version has "transponder"